data/remnant/remnant missions.txt

```diff
@@ -970,6 +970,140 @@ mission "Remnant: Learn Sign Follow Up"
+		conversation
+			`Back on board <ship>, you take a moment to flip through the papers. You quickly realize that little beyond the summary is understandable to someone without a good theoretical understanding of physics. Even with your first-hand experience of wormholes, most of what they are discussing doesn't make much sense.`
+			`	All the academic stuff aside, the summary on the most recent paper confirms that the researchers are making progress: they have determined that the spatial anomaly in the Terminus system is in fact a wormhole with some kind of instability. They also speculate that it may be possible for a ship to artificially stabilize the wormhole enough for it to pass through unharmed.`
+			`	Prefect Torza is definitely going to be interested in seeing this.`
+				accept
+	on complete
+		payment 2000000
+		conversation
+			`The trip back to the Ember Wastes is long, and it gave you plenty of opportunities to think about what this research could lead to. You also wondered about what the Remnant will do, as it seems obvious that their current isolation is about to fail.`
+			`	Torza seems grimly optimistic on receiving your report. "If this had happened without us being aware, it could have gone badly. Fortunately for us, you were in the right place at the right time, and have proven yourself loyal to us. So we are now forewarned and can use this time to plan and prepare for it. Perhaps re-establishing contact with the rest of humanity could even be to our benefit, but that will be for the council to decide.`
+			`	"Thank you, Captain. If you ever come across any other interesting information like this, don't hesitate to bring it to me."`
+
+
+
+# The B mission is for players that kept the Remnant secret from the Deep, but did the wormhole research.
+mission "Remnant: Follow-up on Terminus Researchers B"
+	name "Check up on Wormhole Researchers"
+	description "Return to Bounty to check up on a group of wormhole researchers you met some time ago."
+	minor
+	source
+		government "Remnant"
+	destination "Valhalla"
+	to offer
+		has "Remnant: Learning Sign 2: done"
+		has "Deep: Wormhole Research: offered"
+		not "Remnant: Follow-up on Terminus Researchers A: done"
+		not "Remnant: Follow-up on Terminus Researchers B: active"
+		random < 40
+	on offer
+		conversation
+			`Looking around at the Remnant spaceport, it suddenly strikes you how amazing this is. You never would have imagined that you would end up here back when you earned your pilot's license. It seems so long ago that your biggest concern was deciding whether to pay off your loan by ferrying cargo, doing courier jobs, or escorting merchants. (and on your darker days - contemplating being a pirate).`
+			`	Thinking back, you recall how you helped Ivan and a team of researchers from the Deep study the Ember Waste threshold wormhole. Research like that might be of interest to the Remnant, seeing as it could open the door to research vessels venturing into the Ember Wastes.`
```

Comment:
```suggestion
			`	Thinking back, you recall how you helped Ivan and a team of researchers from the Deep study the Ember Waste threshold wormhole. Research like that might be of interest to the Remnant, seeing as it could open the door to new vessels venturing into the Ember Wastes.`
```
Suggested to minimize usage of `research`.